Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/storage/ExposureSummaryDaoTest.kt

```diff
@@ -23,7 +23,8 @@ class ExposureSummaryDaoTest {
     fun setUp() {
         val context = ApplicationProvider.getApplicationContext<Context>()
         db = Room.inMemoryDatabaseBuilder(
-            context, AppDatabase::class.java
+            context,
```

Comment:
Something seems to have gone wrong with formatting, why are 114 files changed?

Reply:
I'm not sure either. It could be because of the ktlint plugin update to 9.4.1.

Reply:
I doubt that, it looks like some kind of autoformatter having run through the whole project.
We can't merge it like this. Please undo all formatting changes so only the actual changes related to the ticket are left.

Reply:
Ok, maybe. There is some kind of klintformat command, in any case, such huge formatting changes needs to go into a sepperate PR and discussed if we actually want this.

Reply:
Yep, after reverting all the changed files, quickbuild was failing because ktlint complained about the formatting on all of those 100+ files. I've downgraded the plugin and removed the trailing commas and now everything seems fine.